web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -146,63 +148,73 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
 
     const handleSave = useCallback(
         async (values: JobValues) => {
-            if (values.jobType === 'NoExtraction') {
+            const {
+                jobName,
+                jobType,
+                deadline: jobDeadline,
+                categories = [],
+                validationType,
+                owners = [],
+                annotators: jobAnnotator = [],
+                validators: jobValidators = [],
+                annotators_validators = [],
+                is_draft,
+                is_auto_distribution,
+                start_manual_job_automatically,
+                extensive_coverage,
+                selected_taxonomies,
+                pipeline
+            } = values;
+
+            if (jobType === 'NoExtraction') {
                 onRedirectAfterFinish?.();
                 return;
             }
 
+            const deadline = jobDeadline ? new Date(jobDeadline).toISOString() : undefined;
+            const annotators = validationType === 'cross' ? annotators_validators : jobAnnotator;
```

Comment:
Maybe create an enum for validationType and not use magic const? I know that this approach uses in the whole project, but we can start to do on the right way)

Reply:
Yes, this is a good idea for the next pool of the Refazing Rebellion. Thanks!

Reply:
By the way, I just checked this - as far as I can see, we use typing for these constants, so it's definitely not a "magic constant"
![image](https://user-images.githubusercontent.com/50149163/221549091-c08db2c9-1afe-4906-b6f6-0b400a813341.png)
![image](https://user-images.githubusercontent.com/50149163/221549204-168aa8fc-3176-4661-9080-44fdebf1d8d9.png)
![image](https://user-images.githubusercontent.com/50149163/221549413-7d7bf36e-c2a6-43e5-b220-fc430c7d15a7.png)